src/compas/scene/graphobject.py

```diff
@@ -46,11 +46,9 @@ class GraphObject(SceneObject):
     nodecolor = ColorDictAttribute()
     edgecolor = ColorDictAttribute()
 
-    def __init__(self, graph, **kwargs):
-        super(GraphObject, self).__init__(item=graph, **kwargs)
-        self._graph = None
+    def __init__(self, **kwargs):
```

Comment:
this should list `nodecolor`, `edgecolor`, `nodesize`, and not get them from `**kwargs`

Reply:
agree, leftover from the past, will correct all of these in this PR